incubator/citus-postgresql/README.md

```diff
@@ -0,0 +1,200 @@
+# Citus Postgresql Helm Chart
+
+This chart bootstraps a [Citus Postgresql Cluster](https://www.citusdata.com/). Citus is a Postgres extension (not a fork) that intelligently distributes data and queries across nodes so that the database can scale and execute queries fast. The Citus Membership Manager adds automated scaling as well as provisioning functionality.
+
+## Prerequisites
+
+* Kubernetes 1.10.0+
+* Helm 2.10.0+
+
+## Docker Image Source
+
+* [Citus](https://github.com/citusdata/docker)
+* [Citus Membership Mamanger](https://github.com/bakdata/citus-k8s-membership-manager)
```

Comment:
You have a small typo in the link. 

`Mamanger` -> `Manager`

Reply:
Thanks for pointing out, Neil.

Reply:
fixed